### DISCUSSION TITLE
Compatibility with Deep sleep modes

### DISCUSSION THREAD

**Discussion by italicus-web** [sstaub/TickTwo#5] [Q&A]
Hello, 

I have been using this library to put complete an action every 30minutes. 

I'd like to use one deepsleep, is this compatible with ticktwo? 

I assume not but I thought it was worth asking.

**Comment by sstaub** [comment]
I think no. You can use a timer to get out of deepsleep.

**Comment by tochinet** [comment]
deep sleep is not independent of your architecture. So you will need something else than ticker to wake up your system. But you could adapt the library to do that, For AVR (in Arduino UNO etc,) look at https://developerhelp.microchip.com/xwiki/bin/view/products/mcu-mpu/8-bit-avr/structure/sleep/ for ESP32, look at https://randomnerdtutorials.com/esp32-deep-sleep-arduino-ide-wake-up-sources/
